images/capi/hack/boskos.py

```diff
@@ -0,0 +1,143 @@
+#!/usr/bin/env python3
+
+# Copyright 2020 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Checks out a Image-Builder AWS or GCP account"""
+
+import argparse
+import json
+import os
+import time
+
+import requests
+
+BOSKOS_HOST = os.environ.get("BOSKOS_HOST", "boskos")
+BOSKOS_RESOURCE_NAME = os.environ.get('BOSKOS_RESOURCE_NAME')
+
+
+def checkout_account(resource_type, user, cloud, input_state="clean"):
```

Comment:
`clean` was an artifact of an error when we initially created the aws accounts for capa, I don't think we need to keep the support here, especially since the original capa accounts are all now fixed.

```suggestion
def checkout_account(resource_type, user, cloud, input_state="free"):
```